src/newrelic/synthetics.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const request = require('request-promise-native');
+const fs = require('fs');
+const path = require('path');
+
+const MONITOR_FREQUENCY = 15;
+const MONITOR_STATUS = 'ENABLED';
+const MONITOR_THRESHOLD = 7;
+const MONITOR_LOCATIONS = [
+  'AWS_AP_NORTHEAST_1',
+  'AWS_AP_SOUTHEAST_2',
+  'AWS_EU_CENTRAL_1',
+  'AWS_EU_WEST_2',
+  'AWS_SA_EAST_1',
+  'AWS_US_EAST_1',
+  'AWS_US_WEST_1',
```

Comment:
Also (not something you can fix in the PR), but worth discussing: why are there only AWS locations? If there is an AWS<->World connectivity problem, our monitors aren't going to catch it because Runtime is on AWS, too.

Reply:
True, but my understanding is that New Relic deprecated all of the non AWS locations.

Reply:
It's more a question for New Relic than for you.

Reply:
According to NR, all locations are currently AWS, but there's the option to add private minions (not available as a service yet).